### PR TITLE
Hot fix minter-metadata-onchain contract

### PR DESCRIPTION
**Pull request by tasiov**

### CODE REVIEW

File: contracts/minter-metadata-onchain/src/contract.rs [hunk 1]
```diff
@@ -432,32 +431,27 @@ fn _execute_mint(
...
-    if num_mintable_tokens == 0 {
+    let mut mintable_token_ids = MINTABLE_TOKEN_IDS.load(deps.storage)?;
+    if mintable_token_ids.is_empty() {
         return Err(ContractError::SoldOut {});
     }
 
-    let mintable_token_id = match token_id {
+    let mintable_token_position = match token_id {
         Some(token_id) => {
             if token_id == 0 || token_id > config.max_num_tokens {
```
Comment (mcianni) [inline contracts/minter-metadata-onchain/src/contract.rs]: @tasiov Why is token_id == 0 invalid here?

Comment (tasiov) [reply contracts/minter-metadata-onchain/src/contract.rs]: just the token_id indexing starts at 1

Comment (tasiov) [reply contracts/minter-metadata-onchain/src/contract.rs]: that is enforced here:

https://github.com/envadiv/passage-contracts/pull/22/files#diff-6c8cb58d48fcbb6b223e6046abcfcb60bebc73b19ed6c1384530ba5e9b0df780R175

Comment (mcianni) [reply contracts/minter-metadata-onchain/src/contract.rs]: haha, oh, right. makes sense.

**Pull request merged**